files/galaxy/dynamic_job_rules/dev/total_perspective_vortex/vortex_config.yml

```diff
@@ -420,89 +420,3 @@ users:
         params:
           nativeSpecification: "--nodes=1 --ntasks={cores} --ntasks-per-node={cores} --mem={round(mem*1024)} --partition=azuregpu1"
           submit_native_specification: "--nodes=1 --ntasks={cores} --ntasks-per-node={cores} --mem={round(mem*1024)} --partition=azuregpu1"
```

Comment:
@cat-bro Can we override only the partition context variable for this? I'm not sure if that will work, but it should?

Reply:
yes we can. That looks like old-fashioned code